client/src/components/CardDialog/DialogItem/DialogItem.tsx

```diff
@@ -58,18 +70,16 @@ const DialogItem = ({
     switch (item) {
```

Comment:
@street-jackal This function and `chooseIcon` need to convert into React components!